.travis.yml

```diff
@@ -1,10 +0,0 @@
-language: objective-c
-osx_image: xcode9
```

Comment:
Approving this dropping of Xcode 9 CI since it looks like we're dropping support in #82.